cli/Cargo.toml

```diff
@@ -16,7 +16,7 @@ path = "src/main.rs"
 
 [dependencies]
 clap = "2.33.0"
-finch = "0.6"
+finch = { path = "../lib" }
```

Comment:
that's not going to work to publish the cli.
You want to increase the version of finch to 0.7 I guess and have
`finch = { path = "../lib", version = "0.7" }` here after it's released

Reply:
@Keats Thanks. I reverted this change.